### ISSUE TITLE
Separate namespaces for Fedora and Drupal triples

### ISSUE THREAD

**Issue by Natkeeran** [Islandora/documentation#805]
Fedora and Drupal can have some triples in common such as type.  When one issues queries, both additional filters need to be set.  Would it be better if they are in different namespaces?  Or are there use cases where it is better to be in the same namespace?

**Comment by DiegoPino** [comment]
@Natkeeran, you mean namespaces or prefixes? Namespaces are tied to the ontologies your RDF properties belong to. The prefixes that are used are just a convenience to make querying, displaying, etc easier to follow. Also I'm pretty sure, since prefix declaration is like a hint for RDF documents and are applied to RDF properties once displayed (or queried on SPARQL), that there is no way you can have twice the same namespace declared with different prefixes and if you could, the repository or the triple store would not be able to know which predicate should be applied which prefix, ending probably all that belong to the same ontology with the same one (the last one declared). Does that make sense?

Other, more complicated solution would be to manage fedora provided triples v/s drupal ones in different graphs (at triple store leve). But then i guess that would be the ultimate disconnection between our Drupal and fcrepo(means making the existence of the later almost anecdotic in terms of the relevance of their properties.

**Comment by Natkeeran** [comment]
@DiegoPino Sorry, I should have made my question clear.  I mean the triplestore level organization: http://future.islandora.ca:8080/bigdata/#namespaces.  

Ok, I see.  I can see that people would want to drupal and fedora properties.

**Comment by DiegoPino** [comment]
@Natkeeran I mean, that would be possible right. #namespace on that level is actually a graph. So yes, in that case, i would say people could like need that.

**Comment by whikloj** [comment]
Yeah I think I'd agree with @DiegoPino. If you/we insert our triples with the graph specified then you could do
```
select ?s ?p ?o from <#islandora>
```
or 
```
select ?s ?p ?o from <#fedora>
```
or 
```
select ?s ?p ?o 
```
for both

**Comment by DiegoPino** [comment]
++ to both. Reopening!
